src/ctia/properties.clj

```diff
@@ -33,6 +33,7 @@
    (str "ctia.store.es." store ".refresh_interval")  s/Str
    (str "ctia.store.es." store ".replicas") s/Num
    (str "ctia.store.es." store ".shards") s/Num
+   (str "ctia.store.es." store ".default_operator") (s/enum "OR" "AND")
```

Comment:
I don't think we want a different behaviour for each store. IMO, a global is sufficient